tests/Hilke.KineticConvolution.Tests/ConvolutionFactoryTests.cs

```diff
@@ -88,10 +90,9 @@ public void
             // Act
             var actual = _factory.ConvolveArcAndSegment(arc, segment);
 
-
             // Assert
             actual.Should().HaveCount(1);
-            actual.Single().Convolution.As<Segment<double>>().Should().BeEquivalentTo(expected);
+            actual.Single().As<Segment<double>>().Should().BeEquivalentTo(expected);
```

Comment:
tu devrais plutôt faire qqch du genre actual.Single().Should().BeOfType<double>().And.Should()...

Reply:
Done